versionedRouter.js

```diff
@@ -164,21 +172,24 @@ async function compareWithCdkV2(destType, input, flowType, v0Result) {
 }
 
 async function handleV0Destination(destHandler, destType, input, flowType) {
-  const result = {};
+  const v0Result = {};
+  let v0Time = 0;
   try {
-    result.output = await destHandler(input);
-    return result.output;
+    const startTime = Date.now();
+    v0Result.output = await destHandler(input);
+    v0Time = Date.now() - startTime;
+    return v0Result.output;
   } catch (error) {
-    result.error = {
+    v0Result.error = {
       message: error.message,
       statusCode: getErrorStatusCode(error)
     };
     throw error;
   } finally {
```

Comment:
Are we yet to confirm whether to use sync or async modes of calling comparator ?

Reply:
We are waiting for comparator to finish in tests else during tests it will fail and in prod we are not waiting for it.